cli/credentials.ts

```diff
@@ -18,12 +18,17 @@ export function getBigQueryCredentials(): dataform.IBigQuery {
   const cloudCredentials = require(cloudCredentialsPath);
   const locationIndex = selectionQuestion("Enter the location of your datasets:", [
     "US (default)",
-    "EU"
+    "EU",
+    "other"
   ]);
+  let location = locationIndex === 0 ? "US" : "EU";
+  if (locationIndex === 2) {
+    location = question("Enter the location's region name (e.g. 'us-west2'):");
```

Comment:
If I understand correctly, this question is only raised when user chooses `other`, therefore the example given is not very intuitive because it cannot happen that the user now chooses a US or EU region name, perhaps you can change it to:
```
location = question("Enter the location's region name (e.g. 'asia-south1'):");
```

Reply:
so FWIW the user may still want `us-west2` instead of just `US`, but i do think your example is better! fixed.